test/unit/kaocha/private_test.clj

```diff
@@ -0,0 +1,16 @@
+(ns kaocha.private-test
+  (:require [clojure.test :refer :all]
+            [kaocha.testable :as testable]))
+
+(deftest- private-test
```

Comment:
I wonder if we could mark this test and the next one with the `:kaocha/pending` metadata so it doesn't get run directly in the test suite.

Reply:
When seeing `:kaocha/pending`, I thought it is smart because the two tests are not actually required. 

However, as [the document](https://cljdoc.org/d/lambdaisland/kaocha/1.0.861/doc/marking-tests-as-pending) said "Pending tests are tests that are not yet implemented, or that need fixing, and that you don't want to forget about." 

Do you think that if we use `:kaocha/pending` here, we use the semantic correctly?

Reply:
I think this is fine, I actually thought it was pretty smart to do it this way. They're redundant but that shouldn't cause any issues. I wouldn't use `pending` because that will add noise to every test run. Pending is meant as a constant reminder that you have tests that need addressing, that's not really the case here.

Reply:
Good point. I do think including these tests in the same file is a good way to do it. I might introduce a metadata for just our test suite to skip these tests directly.